code-challenges.test.js

```diff
@@ -14,39 +17,152 @@
+        return "1"
+        //else if letter = 'o' return num 0
+      } else if (value === "o" || value === "O") {
+        return "0"
+        //else return the original letter
+      } else {
+        return value
+      }
+    }).join("")
+    }
+
+    // added || value === AEIO in caps...
+    // Pass
+
+
+
+
+
+
+// // --------------------2) Create a function that takes in an array of words and a single letter and returns an array of all the words containing that particular letter.
+
+// // a) Create a test with expects statement using the variable provided.
+
+
+
+describe("returnLetter", () => {
+    const fruitArray = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach", "Kiwi"]
+    
+    const letterA = "a"
+    // Expected output: ["Mango", "Apricot", "Peach"]
+    const letterE = "e"
+    // Expected output: ["Cherry", "Blueberry", "Peach"]
+    it("takes in an array of words and a single letter and returns an array of all the words containing that particular letter", () => {
+        expect(returnLetter(letterA)).toEqual(["Mango", "Apricot", "Peach"])
+        expect(returnLetter(letterE)).toEqual(["Cherry", "Blueberry", "Peach"])
+    })
+  })
+
+  //     ReferenceError: returnLetter is not defined
+
+
+// // b) Create the function that makes the test pass.
+
+// const returnLetter
+const returnLetter = (array, letter) =>{
+  // new array will equal original array, we will use .filter to iterate through the array
+  let newArray = array.filter((word) => {
+    //return the words that contain (includes) specific letter. 
+    //return word().include(letter) ==> originally didnt work because i didnt include .toLowerCase
+  return word.toLowerCase().includes(letter)
+  })
+  // return new array that includes letter
+  return newArray
+}
+
+// pass
+
+// // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a "full house". A full house is exactly one pair and one three of a kind.
+
+// // a) Create a test with expect statements using the variable provided.
+
+describe("fullHouse", () => {
+    const hand1 = [5, 5, 5, 3, 3]
+    // Expected output: true
+    const hand2 = [5, 5, 3, 3, 4]
+    // Expected output: false
+    const hand3 = [5, 5, 5, 5, 4]
+    // Expected output: false
+    const hand4 = [7, 2, 7, 2, 7]
+    // Expected output: true
+    it("Takes in an array of 5 numbers and determines whether or not the array is a full house", () => {
+      expect(fullHouse(hand1)).toEqual(true)
+      expect(fullHouse(hand2)).toEqual(false)
+      expect(fullHouse(hand3)).toEqual(false)
+      expect(fullHouse(hand4)).toEqual(true)
+    })
+    })
+
+    // ReferenceError: fullHouse is not defined 
+
+// // b) Create the function that makes the test pass}
+
+// function = [1,2,3,4,5]
+//if index zero === index 1 and 2, and index 3 === ndex 4
+// else return false
+
+
+
+//i want my function to take in an array of numbers
+const fullHouse = (array) => {
```

Comment:
Awesome work on this problem!!  This one is super tricky, but it looks like you crushed it 👏